scripts/bert/compare_tf_gluon_model.py

```diff
@@ -27,26 +27,33 @@
 import mxnet as mx
```

Comment:
Could you mind creating a folder `convert` and moving 'compare_tf_model.py' and 'convert_tf_model.py' (and some necessary util functions) to that folder?